docs/content/latest/explore/query-1-performance/query-tuning-intro.md

```diff
@@ -0,0 +1,183 @@
+
+{{< note title="Note" >}}
+
+Currently, YugabyteDB does not perform size-specific costing, and assumes every plan generates a thousand rows, which translates to a cost of a thousand. 
+
+{{< /note >}}
+
+To view the execution plan, use the `EXPLAIN` statement. You can use `EXPLAIN` in two modes:
+
+* The default `EXPLAIN` mode shows the plan that the planner created and the predicted cost.
+
+* `EXPLAIN ANALYZE` mode executes the query, measures the time and actual work done, and adds these statistics to the execution plan as 'actual'.
+
+Both the predicted and actual statistics have their uses: 
+
+* The predicted statistics are what the planner uses, and assuming the statistics don't change, the planner will come up with the same execution plan the next time the query is run, even if it caused performance issues in the past.
+
+* The actual statistics show how much and what work was actually performed, and thus are useful to see the effectiveness of the execution plan. 
+
+{{< tip title="Tip" >}}
+
+Complex execution plans can be difficult to read or understand. Use online tools to calculate and visualize execution plan timelines. For example, <https://explain.dalibo.com>.
+
+{{< /tip >}}
+
+## Performance tuning tools in YugabyteDB
+
+### Find slow queries using pg_stat_statements
+
+Use the `pg_stat_statements` extension to get statistics on past queries. Using `pg_stat_statements`, you can investigate queries by userid and dbid, calls, rows, and min, max, mean, standard deviation and total time.
+
+The [`pg_stat_statements`](https://www.postgresql.org/docs/11/pgstatstatements.html) extension module is installed by default, but must be enabled for a database before you can query the `pg_stat_statements` view.
+
+```sql
+CREATE EXTENSION if not exists pg_stat_statements;
+```
+
+To get the output of `pg_stat_statements` in JSON format, visit `https://<yb-tserver-ip>:13000/statements` in your web browser, where `<yb-tserver-ip>` is the IP address of any YB-TServer node in your cluster.
+
+For more information, refer to [Get query statistics using `pg_stat_statements`](../pg-stat-statements).
+
+### View live queries using pg_stat_activity
+
+Use the `pg_stat_activity` view to get information on currently running tasks. Using `pg_stat_activity` you can identify inactive, active, and long time active sessions, and get process information and the current query.
+
+To get the output of `pg_stat_activity` in JSON format, visit `https://<yb-tserver-ip>:13000/rpcz` in your web browser, where `<yb-tserver-ip>` is the IP address of any YB-TServer node in your cluster.
+
+For more information, refer to [Viewing live queries with `pg_stat_activity`](../pg-stat-activity).
+
+### Understand what your queries are doing with EXPLAIN
```

Comment:
There's a bunch of EXPLAIN earlier on the page. Can you consolidate?

Reply:
I could - I was leaving a section on each tool, even though the first 3 are discussed in the earlier overview section, to give each one equal prominence in the page TOC, but they could be removed if you think it's confusing